src/utils/connectors/postgres.py

```
@@ -226,21 +296,64 @@ def decorator(fn):
+                            delay = _get_retry_delay(attempt_number)
+                            reconnect_pool = True
+                            _log_postgres_retry(
+                                fn.__name__, attempt_number, maximum_attempts, delay, error)
+                            continue
                 try:
                     return fn(*args, **kwargs)
                 except (psycopg2.InterfaceError, psycopg2.DatabaseError,
                         psycopg2.pool.PoolError) as error:
-                    logging.error('Database/pool error, retrying: %s', str(error))
+                    if not _is_transient_postgres_error(error):
+                        raise osmo_errors.OSMODatabaseError(f'Error: {str(error)}') from error
```

Ruff (0.16.1)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)